Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM governmentpaas/cf-cli
+
+WORKDIR /usr/src/app
+
+COPY Gemfile Gemfile.lock .ruby-version ./
+
+RUN apk --update --upgrade add build-base openssl && \
+  apk add curl git nodejs && \
+  find / -type f -iname \*.apk-new -delete && \
+  bundle check || bundle install --path vendor/bundle && \
+  rm -rf /var/cache/apk/*
+
+COPY . .
+
+RUN ["npm", "install", "--unsafe-perm"]
```

Comment:
`--unsafe-perrm` ?

Reply:
Was getting errors without this.  Here is some documentation on the issue.

https://stackoverflow.com/questions/18136746/npm-install-failed-with-cannot-run-in-wd